fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/navigation/TabBar.kt

```diff
@@ -64,20 +64,20 @@ fun TabBar(
         ) {
             tabDataList.forEachIndexed { index, tabData ->
                 tabData.selected = index == selectedIndex
-                TabItem(
-                    title = tabData.title,
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .weight(1F),
-                    icon = if (tabData.selected) tabData.selectedIcon else tabData.icon,
-                    textAlignment = tabTextAlignment,
-                    selected = tabData.selected,
-                    onClick = tabData.onClick,
-                    accessory = tabData.badge,
-                    tabItemTokens = tabItemTokens
-                        ?: FluentTheme.controlTokens.tokens[ControlTokens.ControlType.TabItemControlType] as TabItemTokens,
-                    style = if (tabData.selected) FluentStyle.Brand else FluentStyle.Neutral
-                )
+                    TabItem(
```

Comment:
Restore the file if no changes are present. not required to be included in this PR